.github/workflows/buildAndTest.yml

```diff
@@ -22,7 +22,6 @@ concurrency:
 # macos  - arm64  - llvm in-tree     - pytorch binary - build only    # cross compile, can't test arm64
 jobs:
   build-test:
-    continue-on-error: ${{ matrix.torch-version == 'stable' }}
```

Comment:
One concern I have is how making it `Required` will affect the developer workflow for Torch-MLIR contributors. It is cumbersome to locally test Torch-MLIR against two different versions of PyTorch.

Reply:
That's a good question.  I had assumed that the LTC failures were the only ones causing the stable CI to fail, so I imagined that if there are more failures besides LTC, then we'd want to know about the failures and fix them.  @maxbartel, @powderluv do y'all have any suggestions?

Reply:
I think this is only an issue for the next few months and should get better soon after (when 2.1 is stable).  Eventually we can get to version checking in the code.

Reply:
Also lets see how much other stable tests fail and if they do we can revisit this. But for now we can try it out and see if we can make progress.  (@ramiro050 please feel free to un-resolve if you think otherwise).

Reply:
LGTM, we ignored the failures in LTC anyway. Right before a release is also the time where stable and nightly are going to diverge the most, I wouldn't be surprised if we see more failures... Let's see how often this happens. @mgehre-amd FYI